tests/pester-tests/features/PythonSource.Tests.ps1

```diff
@@ -27,12 +27,29 @@ Describe "Python Source" -Tag Chocolatey, UpgradeCommand, PythonSource, ProxySki
         }
 
         It 'Exits with correct exit code (<ExitCode>)' {
-            $Output.ExitCode | Should -Be $ExitCode
+            $Output.ExitCode | Should -Be $ExitCode -Because $Output.String
         }
 
         It 'Outputs properly' {
-            $Output.Lines | Should -Not:($ExitCode -eq 0) -Contain "Alternative sources do not allow the use of the 'all' package name/keyword."
-            $Output.Lines | Should  -Contain "Chocolatey upgraded $Count/$Count packages."
+            $Output.Lines | Should -Not:($ExitCode -eq 0) -Contain "Alternative sources do not allow the use of the 'all' package name/keyword." -Because $Output.String
+            $Output.Lines | Should  -Contain "Chocolatey upgraded $Count/$Count packages." -Because $Output.String
+        }
+    }
+
+    Context "install all" {
+        BeforeAll {
+            # For some reason under kitchen-pester we don't have pip on the path. This might be due to our snapshotting...
+            Import-Module $env:ChocolateyInstall/helpers/ChocolateyProfile.psm1
+            Update-SessionEnvironment

```

Comment:
If I recall from our local vagrant testing, pip wasn't being found on the path there, but these tests didn't need it to be because pip is never attempted to be run (I think python3 is still needed to be installed because Chocolatey CLI will require the package, but I seem to recall the path doesn't matter here).
```suggestion
```